app/Import/AudioImport.php

```diff
@@ -154,6 +154,11 @@ public function validate()
           && !empty($row[$key]) && !$this->valueExists(AudioItem::class, 'track_configuration', $row[$key])) {
           $bag->add($key, $key . ' must already exist in the database.');
         }
+        // Validate track configuration exists in the DB
+        if ($key === 'Base'
+          && !empty($row[$key]) && !$this->valueExists(AudioItem::class, 'base', $row[$key])) {
+          $bag->add($key, $key . ' must already exist in the database.');
+        }
```

Comment:
Doesn't need to be done in this PR, but could this code be DRY-ed up a bit? Looks mostly the same as the above couple of validations.

Reply:
Hmm, actually, I guess it already was partly DRY-ed in the last PR I looked at, wasn't it. Still could be cleaned up a bit more with the similar check and messaging I suppose.

Reply:
Sure, that's a good point. I'll work on it as part of this PR since it's not a rush.

Reply:
Ok, the DRY-ed up code is up, can you take another look?